src/bindings/python/tests/test_runtime/test_compiled_model.py

```diff
@@ -54,6 +57,31 @@ def test_export_import(device):
     assert np.argmax(res[new_compiled.outputs[0]]) == 531
 
 
+@pytest.mark.skipif(
+    condition=sys.version_info >= (3, 12),
+    reason="Fails on any Linux platform with Python 3.12. Ticket CVS-133903",
+)
+def test_export_import_with_crypto(device):
+    core = Core()
+
+    if props.device.Capability.EXPORT_IMPORT not in core.get_property(device, props.device.capabilities):
+        pytest.skip(f"{core.get_property(device, props.device.full_name)} plugin due-to export, import model API isn't implemented.")
+
+    config = {}
+    config["CACHE_CRYPTO_CALLBACK"] = [encrypt_base64, decrypt_base64]
```

Comment:
why ov.props is not used here instead of hardcoded string?

Reply:
updated.

Reply:
@ilya-lavrenov I tested that props.cache_crypto_callback is a write-only property and can not be used as a string, so I restore the hardcoded string